src/weather.js

```diff
@@ -14,9 +14,18 @@ export class Weather {
 
   return  this.celsius;
 
+}
 
+async londonDescription() {
+  const response = await fetch('https://api.openweathermap.org/data/2.5/weather?q=London'+ '&appid='+process.env.API_KEY)
+  const json = await response.json();
+  const weatherDescription = json.weather[0].description;
+console.log(weatherDescription);
+  return weatherDescription;
 }
-  convert(temp) {
+
+
+convert(temp) {
```

Comment:
Is the API call getting the temperature in Celsius? If so, should be able to revert the Kelvin conversion method.

Reply:
Changed API thank you